dataactvalidator/config/sqlrules/d32_detached_award_financial_assistance_1.sql

```diff
@@ -0,0 +1,17 @@
+-- PeriodOfPerformanceStartDate is an optional field, but when provided, must follow YYYYMMDD format
+CREATE OR REPLACE function pg_temp.is_date(str text) returns boolean AS $$
+BEGIN
+  perform CAST(str AS DATE);
+  return TRUE;
+EXCEPTION WHEN others THEN
+  return FALSE;
+END;
+$$ LANGUAGE plpgsql;
+
+SELECT
+    row_number,
+    period_of_performance_star
+FROM detached_award_financial_assistance
+WHERE submission_id = {0}
+    AND period_of_performance_star IS NOT NULL
```

Comment:
Ok cool. If the broker inserts it properly when uploaded we may be good. But I know that for the unit tests, we can force a blank value so not sure how we want to handle that in the SQL